virtualmachine/aws/vm.go

```diff
@@ -383,10 +383,18 @@ func (vm *VM) ResetKeyPair() {
 	vm.KeyPair = ""
 }
 
-func (vm *VM) AddDisk() error {
-        return errors.New("TO-DO")
+// AddDisk adds disk to instance
```

Comment:
This code of dummy methods can be removed now. And added with implementation only.